model/Mastermind.php

```diff
@@ -183,4 +216,50 @@ public function decrementRemainingAttempts(): void
         $this->remainingAttempts--;
     }
 
+    /**
+     * @param Combination $proposition
+     * @throws Exception Si la partie n'existe pas ou qu'elle est terminée
+     */
+    public function play(Combination $proposition): void
+    {
+        if ($this->remainingAttempts <= 0) {
+            throw new Exception('Partie non créée');
+        } else {
+            $this->addProposition($proposition);
```

Comment:
C'est mieux d'utiliser directement `$this->propositions[] - $proposition;`

Reply:
Tu veux dire que : $this->propositions[] = $proposition;
remplace $this->addProposition($proposition);
C'est bien ça ?

Reply:
Vu ce qu'il y a en dessous, c'est bien ça. :)
ça marche.

Reply:
Oui c'est ça, mon doigt a du ripper ^^